src/app/speakers/speakers/speakers.component.scss

```diff
@@ -117,16 +122,43 @@
   }
 }
 
+.grid-item:hover .speakerData .background {
+  opacity: 0;
+  transition: opacity 1.2s;
+}
+
 .speakerData {
-  color: white;
-  bottom: 10px;
+  color: #fff;
   font-size: 6vw;
-  left: 10px;
   position: absolute;
   z-index: 5;
+  padding: 25px 15px 10px;
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Why this line, i thought the style `* { box-sizing: border-box; }` was in place, was it deleted or overwritten?

Reply:
didn't take the time to track that this specific style was already applied. Deleting in next PR